jsk_unitree_robot/cross/repos/unitree.repos

```diff
@@ -21,3 +21,11 @@ repositories:
     type: git
     url: https://github.com/Affonso-Gui/app_manager
     version: enable-parallel-run-apps
+  google_chat_ros:
```

Comment:
please add link to PR of this branch, I also forget to add https://github.com/PR2/app_manager/pull/59 to app_manager section. Please added them to to Line 20, thnak you.

Reply:
Thank you.
https://github.com/k-okada/jsk_robot/pull/66/commits/42283bb4366816dce8c53b2d0ea3589e5b179cbf
I added the comments to add reference PR.